api/endpoint-tests/apds/get.js

```diff
@@ -4,19 +4,21 @@ const {
   getFullPath,
   login,
   request,
-  unauthenticatedTest
+  unauthenticatedTest,
+  unauthorizedTest
 } = require('../utils');
 
 tap.test('APD endpoint | GET /apds', async getAPDsTest => {
   const url = getFullPath('/apds');
   await db().seed.run();
 
   unauthenticatedTest('get', url, getAPDsTest);
+  unauthorizedTest('get', url, getAPDsTest);
 
   getAPDsTest.test(
-    'when authenticated as a user with a state',
+    'when authenticated as a user without a state',
     async invalidTest => {
-      const cookies = await login();
+      const cookies = await login('all-permissions-no-state', 'password');
```

Comment:
Renamed this user to be more descriptive.  Hopefully it's obvious what this user represents now. 😄